src/rest/app.py

```diff
@@ -207,12 +208,8 @@ async def translate(
     transcription: TranscriptionData = Body(...),
 ):
     """Translate text to a target language."""
-    # TODO: Here the supported language codes might differ
-    if target_language not in config["supported_language_codes"]:
-        raise HTTPException(
-            status_code=400,
-            detail=f"Unsupported language code: {target_language}",
-        )
+    check_language_supported(target_language)
```

Comment:
I personally find this a bit confusing that this can throw an exception but there is no indication that this stops the execution flow. I feel like stopping the request handling should not be a side effect.
My suggestions:
1. Make the function return a bool and raise the exception right here
2. Rename the function to `language_supported_guard` or something. Possibly this could also be a decorator but I am not sure

Reply:
yes I also thought for the 2nd solution, would go with that solely and add a comment